### ISSUE TITLE
Seeing multiple "reaped unknown child process" messages

### ISSUE THREAD

**Issue by anson2590** [puma/puma#3419]
When we run application we are seeing multiple logs like "! reaped unknown child process pid=34 status=pid 34 exit 0" is there a way we can disable this log in our application ?

**To Reproduce**
- When the application runs on docker in splunk logs we are seeing lot of above of mentioned error message.

**Comment by dentarg** [comment]
It comes from  https://github.com/puma/puma/blob/a7e3d944fa66bbba355abb9efed8ce53048c7e7c/lib/puma/cluster.rb#L587

It was added in https://github.com/puma/puma/pull/3255

You can't disable this logging

**Comment by dentarg** [comment]
I think you want to know that this is happening and adjust your application for it to not happen. @byroot what do you think?

**Comment by byroot** [comment]
Agreed. This mean your application is spawning sub processes but not reaping them, which means you likely have lots of zombie processes.

I'm not against adding a way to disable that log, but it will just hide the problem, not fix it. I'd suggest to at least have a look into what these processes are.

**Comment by dentarg** [comment]
Thanks Jean. I'll convert this to a discussion for now.
